src/app/config.js

```diff
@@ -0,0 +1,94 @@
+/* jshint maxlen:false */
+define([
+    'dojo/has',
+    'dojo/request/xhr',
+
+    'esri/config'
+], function (
+    has,
+    xhr,
+
+    esriConfig
+) {
+    // force api to use CORS on mapserv thus removing the test request on app load
+    // e.g. http://mapserv.utah.gov/ArcGIS/rest/info?f=json
+    esriConfig.defaults.io.corsEnabledServers.push('mapserv.utah.gov');
+    esriConfig.defaults.io.corsEnabledServers.push('basemaps.utah.gov');
+    esriConfig.defaults.io.corsEnabledServers.push('api.mapserv.utah.gov');
+
+    window.AGRC = {
+        // errorLogger: ijit.modules.ErrorLogger
+        errorLogger: null,
+
+        // app: app.App
+        //      global reference to App
+        app: null,
+
+        // version.: String
+        //      The version number.
+        version: '0.1.1',
+
+        // apiKey: String
+        //      The api key used for services on api.mapserv.utah.gov
+        apiKey: '', // acquire at developer.mapserv.utah.gov
+
+        // exportWebMapUrl: String
+        //      print task url
+        exportWebMapUrl: 'http://mapserv.utah.gov/arcgis/rest/services/Utilities/PrintingTools/GPServer/Export%20Web%20Map%20Task',
```

Comment:
This property is not used in the app and should be removed. Looks like there are a few more that need to be cleaned up in this object.